src/CMakeLists.txt

```diff
@@ -132,24 +128,29 @@ else(WITH_CUDA)
     set_target_properties(CCTag PROPERTIES VERSION ${PROJECT_VERSION})
   endif( COMPILE_STATIC_CCTAG_LIBRARY )
 
-  target_include_directories(CCTag 
+endif(WITH_CUDA)
+
+target_include_directories(CCTag 
                            PUBLIC "$<BUILD_INTERFACE:${CMAKE_CURRENT_LIST_DIR}>"
                                   "$<INSTALL_INTERFACE:$<INSTALL_PREFIX>/include>"
                            PUBLIC ${Boost_INCLUDE_DIRS} ${Eigen_INCLUDE_DIR} ${OpenCV_INCLUDE_DIRS}
-                           PRIVATE ${TBB_INCLUDE_DIR} ${PNG_INCLUDE_DIRS} ${JPEG_INCLUDE_DIR}) 
+                           PRIVATE ${TBB_INCLUDE_DIRS} ${JPEG_INCLUDE_DIR}) 
 
-  get_target_property(testprop CCTag INTERFACE_INCLUDE_DIRECTORIES )
-  message(STATUS "testprop: ${testprop}")
+get_target_property(testprop CCTag INTERFACE_INCLUDE_DIRECTORIES )
+message(STATUS "testprop: ${testprop}")
 
-  target_compile_definitions(CCTag 
+target_compile_definitions(CCTag 
                            PUBLIC ${Boost_DEFINITIONS} 
-                           PRIVATE ${TBB_DEFINITIONS} ${PNG_DEFINITIONS}) 
+                           PRIVATE ${TBB_DEFINITIONS}) 
   
-  target_link_libraries(CCTag 
+target_link_libraries(CCTag 
                       PUBLIC ${OpenCV_LIBS} ${Boost_LIBRARIES} ${Eigen_LIBRARIES} 
-                      PRIVATE ${TBB_LIBRARIES} ${PNG_LIBRARIES} ${JPEG_LIBRARIES} pthread dl)
+                      PRIVATE ${TBB_LIBRARIES} ${JPEG_LIBRARIES} pthread dl)
 
-endif(WITH_CUDA)
+if(HAVE_PNG)
```

Comment:
maybe `PNG_FOUND` ? I can't see any definition for `HAVE_PNG`
Also, I suspect that PNG is only needed by opencv, if it is compiled with. It seems to me that the png library it is not explicitly used anywhere in the code
And we can safely remove these two lines
https://github.com/alicevision/CCTag/blob/develop/src/applications/simulation/main.cpp#L8

@griwodz